sbpy/data/names.py

```diff
@@ -342,7 +342,7 @@ def parse_comet(s):
                 if len(el[0]) > 0:
                     typnumber = el[0].replace('/', '')
                     try:
-                        r['type'] = re.findall('[PDCXAI]', typnumber)[0]
+                        r['type'] = re.findall('[PDCXI]', typnumber)[0]
```

Comment:
Looks like the type "I" is included here, but not matched in the pattern in line 311.  Have we decided whether and how to include interstellar objects?  I think we should: 1) remove "I" here if it's not implemented to avoid confusion in the code, and 2) mention in the document that interstellar objects are not parsed at this time, but will be implemented later.

Reply:
Good catch.  I've removed the I here.  I have the documentation addressed in #337 : https://sbpy--337.org.readthedocs.build/en/337/sbpy/data/names.html